skatepark/block-templates/index.html

```diff
@@ -0,0 +1,32 @@
+<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
```

Comment:
Should we bring the header / footer parts in the same PR? Just asking because it errors currently with "template part not found"

Reply:
honestly, I was going to just have the post-template block and leave it like that, the index needs to be re-done completely and it has its own separate issue, I just didn't want to have it be empty at all.